src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,67 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type TodoModalProps = {
+  user: User | null,
+  onModalClose: (id?: number) => void,
+  todo: Todo | undefined;
+};
+
+export const TodoModal: React.FC<TodoModalProps> = ({
+  user,
+  onModalClose,
+  todo,
+}) => {
+  const email = user?.email;
+  const name = user?.name;
+
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
+      {user ? (
         <div className="modal-card">
           <header className="modal-card-head">
             <div
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => onModalClose()}
```

Comment:
No need one more callback
```suggestion
              onClick={onModalClose}
```